kolla/defaults.yml

```diff
@@ -12,7 +12,11 @@ kolla_install_type: source
 # Remove this if pulling CentOS8 containers
 openstack_tag_suffix: ''
 
+# Set according to https://docs.openstack.org/kolla-ansible/latest/user/virtual-environments.html#target-hosts
+# virtualenv specifies the path to the kolla virtualenv, while ansible_python_interpreter must be set explicitly
+# since it cannot be templated. This is overridden during bootstrap by cc-ansible.
 virtualenv: /etc/ansible/venv
+ansible_python_interpreter: /etc/ansible/venv/bin/python
```

Comment:
Good comment about templating not working, I forgot about that, but it makes sense.